metrics/web/test/dashboard/presentation/widgets/build_result_bar_test.dart

```diff
@@ -121,18 +133,99 @@ void main() {
     testWidgets(
       "applies the minimum height to the metrics colored bar from the constraints",
       (tester) async {
-        const expectedHeight = 10.0;
-
         await tester.pumpWidget(
           _BuildResultBarTestbed(
-            constraints: const BoxConstraints(minHeight: expectedHeight),
+            constraints: const BoxConstraints(minHeight: height),
             buildResult: successfulBuildResult,
           ),
         );
 
         final bar = getMetricsColoredBar(tester);
 
-        expect(bar.height, equals(expectedHeight));
+        expect(bar.height, equals(height));
+      },
+    );
+
+    testWidgets(
+      "displays the metrics colored bar with the build status from the build result view model",
+      (tester) async {
+        final expectedStatus = successfulBuildResult.buildStatus;
+        await tester.pumpWidget(_BuildResultBarTestbed(
+          buildResult: successfulBuildResult,
+        ));
+
+        final bar = getMetricsColoredBar(tester);
+
+        expect(bar.value, equals(expectedStatus));
+      },
+    );
+
+    testWidgets(
+      "displays the metrics animated bar if the given build result view model is in-progress",
```

Comment:
```suggestion
      "displays the in-progress animated bar if the given build result view model is in progress",
```
Please, apply this suggestion to all similar situations below.